src/nodes.jl

```diff
@@ -616,7 +616,8 @@ Print a simplified string representation of the `CollectNode` with its nodes Vec
 result DeferredFuture RemoteChannel parameters, and its label.
 """
 function Base.show(io::IO, node::CollectNode)
-    print(io, "$(typeof(node).name.name)($(node.nodes),$(node.result),\"$(node.label)\")")
+    nodes = "DispatchNode[$(join(node.nodes, ","))]"
```

Comment:
You don't want to add it inside the string, you want to print to `io` as you go along:

```julia
print(io, "DispatchNode[")
join(io, node.nodes, ",")
print(io, "]"
```